src/app/modules/app-routing.module.ts

```diff
@@ -2,8 +2,20 @@ import { NgModule } from '@angular/core';
 import type { Routes } from '@angular/router';
 import { RouterModule } from '@angular/router';
 import type { CartographyFeatureModule } from '@features/cartography';
+import { ContentLayout } from '../pages/layouts/content/content.layout';
+import { HomePage } from '../pages/home/home.page';
 
 const ROUTES: Routes = [
+  {
+    children: [
+      {
+        component: HomePage,
+        path: ''
+      }
+    ],
+    component: ContentLayout,
+    path: 'accueil'
```

Comment:
```suggestion
    path: ''
```
C'est mieux d'essayer de visualiser les chemins comme un arbre dont la page principale est la raçine.